src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
+
+
+
+
+
+class Content extends Component{
+    constructor(props){
+        super(props);
+        this.state={};
+
+        this.handlename=this.handlename.bind(this);
+    }
+
+
+
+    handlename(props){
+        alert(renderboth)
+    }
+
+
+    render(props){
+        return(
+            <div className="container">
+                <div className="row">
+                    <div className="col-md-12 offset-4">
+                        <h1>Hello {}</h1>
```

Comment:
why is there an object curly bracket in there?
```suggestion
                        <h1>Hello</h1>
```